### PR TITLE
Count quota

### PR DESCRIPTION
**Pull request by henrysun007**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

It is necessary to count the function invocation times. Almost every FaaS platform has this feature. A limit to the times is also added in this PR.  

Fixes # (issue)

## Type of change (select or add applied and delete the others)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API change with a documentation update
- [ ] Additional test coverage
- [ ] Code cleanup or just sync with upstream third-party crates

## Checklist

- [X] Fork the repo and create your branch from `master`.
- [X] If you've added code that should be tested, add tests.
- [ ] If you've changed APIs, update the documentation.
- [X] Ensure the tests pass (see CI results).
- [X] Make sure your code lints/format.

### CODE REVIEW

**Review by mssun — DISMISSED** [review]
LGTM. Thanks!

**Review by bigdata-memory — DISMISSED** [review]
thanks.

File: sdk/rust/src/lib.rs [hunk 1]
```diff
@@ -233,6 +236,15 @@ impl FrontendClient {
         if let Some(outputs) = outputs {
             builder = builder.outputs(outputs);
         }
+        if let Some(usage_quota) = usage_quota {
+            let usage_quota = if usage_quota < 0 {
```
Comment (volcano0dr) [inline sdk/rust/src/lib.rs:240]: `let usage_quota = (usage_quota >= 0).then_some(usage_quota);`

File: services/proto/src/teaclave_frontend_service.rs [hunk 2]
```diff
@@ -1122,6 +1156,11 @@ impl std::convert::TryFrom<proto::RegisterFunctionRequest> for RegisterFunctionR
             .into_iter()
             .map(FunctionArgument::try_from)
             .collect();
+        let usage_quota = if proto.usage_quota < 0 {
```
Comment (volcano0dr) [inline services/proto/src/teaclave_frontend_service.rs:1159]: `let usage_quota = (proto.usage_quota >=0).then_some(proto.usage_quota)`

File: services/proto/src/teaclave_frontend_service.rs [hunk 3]
```diff
@@ -1298,6 +1346,47 @@ impl From<GetFunctionRequest> for proto::GetFunctionRequest {
...
+        }
+    }
+}
+
+impl std::convert::TryFrom<proto::GetFunctionUsageStatsRequest> for GetFunctionUsageStatsRequest {
+    type Error = Error;
+
+    fn try_from(proto: proto::GetFunctionUsageStatsRequest) -> Result<Self> {
+        let function_id: ExternalID = proto.function_id.try_into()?;
+        let ret = Self { function_id };
```
Comment (volcano0dr) [inline services/proto/src/teaclave_frontend_service.rs:1362]:
```
Ok(Self {
    function_id
})
```

**Review by volcano0dr — APPROVED** [review]
LGTM!

**Pull request merged**